debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -252,4 +279,14 @@ private static int validateCollectionBlacklist(Configuration config, Field field
         }
         return 0;
     }
+    
+    private static int validateDatabaseBlacklist(Configuration config, Field field, ValidationOutput problems) {
```

Comment:
I think it is a good plan to separate this issue out and look at it holistically cross all connectors.